<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QStackedWidget" name="stw1">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTreeWidget" name="appsListW">
            <property name="columnCount">
             <number>3</number>
            </property>
            <column>
             <property name="text">
              <string>Application</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Arguments</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>PID</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2"/>
       </widget>
       <widget class="QStackedWidget" name="stw2">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="page_3">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTreeWidget" name="widgetListW">
            <column>
             <property name="text">
              <string>Object Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Widget Type</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_4">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTreeWidget" name="twThreads">
            <column>
             <property name="text">
              <string>Device Threads</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_7">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0" colspan="3">
           <widget class="QTreeWidget" name="twActions">
            <column>
             <property name="text">
              <string>TAction</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Mode</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="cbActionAutoRefresh">
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string>Auto Refresh</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QSpinBox" name="sbActRefreshPeriod">
            <property name="minimum">
             <number>10</number>
            </property>
            <property name="maximum">
             <number>9998</number>
            </property>
            <property name="singleStep">
             <number>100</number>
            </property>
            <property name="value">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>period [ms]:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QStackedWidget" name="stw3">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_5">
        <widget class="QWidget" name="">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>9</y>
           <width>901</width>
           <height>311</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QSplitter" name="splitter_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>3</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QTreeWidget" name="slotsListW">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <column>
              <property name="text">
               <string>Methods (slots)</string>
              </property>
             </column>
            </widget>
            <widget class="QTreeWidget" name="propertiesListW">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <column>
              <property name="text">
               <string>Properties</string>
              </property>
             </column>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Execute</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Arg 1:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="leArg1"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Arg 2:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="leArg2"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Arg 3:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="leArg3"/>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Arg 4:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="leArg4"/>
             </item>
             <item row="4" column="0">
              <widget class="QPushButton" name="pbRead">
               <property name="text">
                <string>Read</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QPushButton" name="pbExecute">
               <property name="text">
                <string>Execute</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Return value:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLineEdit" name="leReturnValue"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <zorder>slotsListW</zorder>
        <zorder>propertiesListW</zorder>
        <zorder>groupBox</zorder>
        <zorder>groupBox</zorder>
       </widget>
       <widget class="QWidget" name="page_8"/>
       <widget class="QWidget" name="page_6">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0" rowspan="3">
          <widget class="QScrollArea" name="scrollArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>396</width>
              <height>366</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:7pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Target execution events from &lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;all&lt;/span&gt; applications of this &lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;session&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QTreeWidget" name="twTargetExecutionEvents">
           <property name="toolTip">
            <string>This widget receives signals from all TActions in the current section!</string>
           </property>
           <column>
            <property name="text">
             <string>Application</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Host:display</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Target</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Date Time</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Success</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Exception</string>
            </property>
           </column>
          </widget>
         </item>
         <item row="2" column="1">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>395</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="pbClearEvents">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>929</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
